recipes/go-precompiled/build-core.bat

```diff
@@ -14,6 +14,9 @@ call "%PREFIX%\etc\conda\activate.d\activate_z60-go.bat"
 mkdir "%PREFIX%\go"
 xcopy /s /y /i /q "%SRC_DIR%\go\*" "%PREFIX%\go\"
 
+rem Remove Invalid UTF-8 Filename
+RMDIR /S /Q "%PREFIX%\go\test\fixedbugs\issue27836.dir
```

Comment:
We remove this directory because of a bug on conda-package-handling. Here is the traceback:
```python-traceback
INFO :: Time taken to mark (prefix)
        0 replacements in 0 files was 3.99 seconds
Traceback (most recent call last):
  File "C:\Miniconda\lib\site-packages\conda_package_handling\api.py", line 82, in create
    out = SUPPORTED_EXTENSIONS[ext].create(prefix, file_list, out_fn, out_folder, **kw)
  File "C:\Miniconda\lib\site-packages\conda_package_handling\tarball.py", line 145, in create
    '.tar.bz2', 'bzip2')
  File "C:\Miniconda\lib\site-packages\conda_package_handling\tarball.py", line 78, in create_compressed_tarball
    _create_libarchive(fullpath, files, compression_filter, filter_opts)
  File "C:\Miniconda\lib\site-packages\conda_package_handling\tarball.py", line 64, in _create_libarchive
    archive_utils.create_archive(fullpath, files, compression_filter, filter_opts)
  File "C:\Miniconda\lib\site-packages\conda_package_handling\archive_utils.py", line 29, in create_archive
    raise ArchiveCreationError(message)
conda_package_handling.exceptions.ArchiveCreationError: \??\C:\Miniconda: Couldn't visit directory while writing file: go\test\fixedbugs\issue27836.dir\ï¿½foo.go
```